must-gather/collection-scripts/gather_elasticsearch_operator_resources

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
this should likely live in the EO's own must-gather and CLO could call into that

Reply:
While the separation would be nice, I don't see user's separating EO from CLO in this respect; it's just logging to them.  EO is not an independent product so we should just point user's to CLO and make a change in the future if EO ever truly is more independent.

Reply:
so what about the Jaeger case?

Reply:
I favour for now a cluster-logging `must-gather` that encompasses the whole stack in one single command. This simplifies retiring logging-dump.sh for the start with more recognizable alternative. IMHO we can re-think what is a good must-gather for an independent elasticsearch-operator later. A little bit of overlap between both shouldn't hurt.

Regarding the jaeger case, I would say there should a jaeger must-gather for the tracing stack than encompasses elasticsearch resources the way they use them.

Reply:
thats fine, i personally don't think its a little overlap between what jaeger would want and CLO would want (regarding EO). if we think about it as "a customer has jaeger installed but has an issue with EO" i feel we would want to control what sort of information is gathered as part of troubleshooting it. yes, the jaeger team can always add more on top of that, but my original point was more about ensuring the things that are specific to debugging EO on its own should live in an EO provided `must-gather`:

* install plans
* CSVs
* CRs
* EO logs
* ES logs

Reply:
Let's be clear, at the moment there are no EO-only customer's only.  We are not supporting an independent use of EO outside the logging and jaeger use case.  If we ever get to that point then we will make that change